libdnf/conf/ConfigMain.cpp

```diff
@@ -181,8 +184,7 @@ class ConfigMain::Impl {
     OptionString logdir{"/var/log"};
     OptionNumber<std::int32_t> log_size{1024 * 1024, strToBytes};
     OptionNumber<std::int32_t> log_rotate{4, 0};
-    // More important varsdirs must be on the end of vector
-    OptionStringList varsdir{std::vector<std::string>{"/etc/yum/vars", "/etc/dnf/vars"}};
+    OptionStringList varsdir{VARS_DIRS};
```

Comment:
Why is this being turned into a constant?

Reply:
Because the same default is used on two places now. In `ConfigMain::Impl()` and in `dnf_context_init()`. I prefer to have definition on one place.